apartment.rb

```diff
@@ -1,2 +1,29 @@
 class Apartment
+
+  attr_reader :number
+  attr_accessor :rooms, :monthly_rent
+
+  def initialize(args={})
+    @number = args.fetch(:number, "")
+    @rooms = args.fetch(:rooms, "")
+    @monthly_rent = args.fetch(:monthly_rent, 1000)
+  end
+
+  def total_sqft
+    area = @rooms.map { |room| room.sqft }
```

Comment:
Another way to solve this is by just using the reduce function.
```
@rooms.reduce(0) { |sum, room| sum += room.sqft }
```
0 is the starting value and the block takes each rooms sqft and adds it to the sum value. The return value is the sum. @JWong411